getting-started/proc-adding-github-to-frontend.adoc

```diff
@@ -0,0 +1,15 @@
+[id"proc-adding-github-to-frontend"]
+
+= Adding the GitHub provider to the {product-short} frontend​
+
+To add the provider to the frontend, add the sign in configuration to your `app-config.yaml` file. For example: 
+
+[source,yaml]
+----
+signInPage: github
+----
+
+.Additional resources
+
+* For information about authenticating Backstage access with GitHub, see link:https://backstage.io/docs/integrations/github/github-apps/[GitHub Authentication Provider] in the community documentation. 
+* For information about adding the provider to the Backstage frontend, see link:https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/getting-started.md[Enabling authentication in Showcase] in the community documentation. 
```

Comment:
Please remove the upstream link from the documentation. We do not add upstream links to our enterprise docs.

Reply:
SME says that it is necessary to include this documentation, so I am leaving the links for now, but I responded in a comment that I would like to discuss this post-GA to get Docs and Eng on the same page about including/referencing upstream docs in RHDH docs.